src/components/FloatingHelper/HelperContent/HelperContent.tsx

```diff
@@ -46,28 +48,31 @@ export const HelperContent: React.SFC<HelperContentProps> = (
             </Text>
           </div>
         )}
-        {actionText &&
+        {actionText && onAction &&
```

Comment:
so you want the text to be invisible when `actionText="something` but  `onAction` is not given?

Reply:
I was trying to follow WSR's `SectionHelper` behavior:
https://github.com/wix/wix-style-react/blob/b5a7c246265475cd799dd1420780e45aad0c237f/src/SectionHelper/index.js#L53

It is documented.

We can also add a propType validation that will give a proper warning message.

Another solution would be to create a new component (-:
`<ActionButton>` - which will have both `text` and `onClick` props as required props.